src/textual/widget.py

```diff
@@ -812,13 +830,22 @@ def layout(self) -> Layout:
 
     @property
     def is_container(self) -> bool:
-        """Check if this widget is a container (contains other widgets)
+        """Check if this widget is a container (contains other widgets).
 
         Returns:
             bool: True if this widget is a container.
         """
         return self.styles.layout is not None or bool(self.children)
 
+    @property
+    def is_scrollable(self) -> bool:
+        """Check if this Widget may be scrolled.
+
+        Returns:
+            bool: True if this widget may be scrolled.
+        """
+        return self.is_container
```

Comment:
No, they are distinct but related concepts. A container is a widget with children. When that is scrolled, the compositor adjusts the position of the children on screen.

A ScrollView is not a container, in that it has no children. But it can have scrollbars. A Scrollview updates its own region and doesn't rely on the compositor to scroll.

Make sense?